docs/markdown/authoritative/backend-generic-sql.md

```diff
@@ -100,16 +100,17 @@ this implies that a slaved domain will not stay disabled.
 
 ## Autoserial
 The autoserial functionality makes PowerDNS generate the SOA serial when the SOA
-serial set to `0` in the database. The serial in SOA responses is set to the
-highest value of the `change_date` field in the "records" table.
+serial set to `0` in the database. The serial in SOA responses is set to what's
+provided by `zone-lastchange-query`. By default, this is the highest value of the
+`change_date` field in the "records" table).
 
 
 # Handling DNSSEC signed zones
 To enable DNSSEC processing, the `backend-dnssec` option must be set to 'yes'.
 
 ## Rules for filling out DNSSEC fields
 Two additional fields in the 'records' table are important: 'auth' and 'ordername'.
-These fields are set correctly on an incoming zone transfer, and also by running
+These fields are set correctly on a zone transfer, and also by running
```

Comment:
the fields are only set during an incoming axfr, so the original text is more accurate